pkgs/tools/misc/coreutils/default.nix

```diff
@@ -16,7 +16,7 @@
 , minimal ? true
 , withOpenssl ? !minimal, openssl
 , withPrefix ? false
-, singleBinary ? "symlinks" # you can also pass "shebangs" or false
+, singleBinary ? if stdenv.isFreeBSD then false else "symlinks" # you can also pass "shebangs" or false
```

Comment:
Link to upstream issue, either here or in commit message?

Reply:
Is there a coreutils bug tracker other than the mailing list? I emailed them like 12h ago and it still hasn't shown up in their archives for a link.

Reply:
To bug-coreutils@gnu.org?  Did you send it in plain text?

Reply:
I did

Reply:
I guess it probably got held for moderation/spam review then?  You could mention the message ID and date of the message, and then if it ever does show up it'll be findable.

Reply:
Looks like it got approved overnight. It's in the commit message now.